### ISSUE TITLE
what does predict_img in predict.py return

### ISSUE THREAD

**Issue by lightersss** [milesial/Pytorch-UNet#232]
It looks like a matrix consists of False or True ,not the float or long number
```
def predict_img()
#somthings 
    return full_mask > out_threshold
```

**Comment by milesial** [comment]
It is indeed the binary classes after applying a threshold. You can remove the threshold if you want the raw sigmoid/softmax output.
